lib/internal/fs/streams.js

```diff
@@ -92,6 +93,9 @@ const FileHandleOperations = (handle) => {
       PromisePrototypeThen(handle.close(),
                            () => cb(), cb);
     },
+    fsync: (fd, cb) => {
+      PromisePrototypeThen(handle.sync(), () => cb(), cb);
```

Comment:
Is this line correct?

```suggestion
      PromisePrototypeThen(handle.fsync(), () => cb(), cb);
```

Reply:
I don't know why it was named that way, but https://nodejs.org/api/fs.html#filehandlesync